### ISSUE TITLE
// tslint:disable-next-line:max-line-length not working.

### ISSUE THREAD

**Issue by harshi1991** [palantir/tslint#3388]
### Bug Report

// tslint:disable not working 

- __TSLint version__: 5.8.0
- __TypeScript version__: 2.3.4
- __Running TSLint via__: CLI 

#### TypeScript code being linted

```ts
// code snippet
// tslint:disable-next-line:max-line-length
import { LocationSpecificSurveyInput } from '../../dashboard/servicesComponets/surveys/locationSpecificSurvey/location-specific-survey.model';

```

#### Actual behavior
ERROR: src/app/searchResults/locationAbout/location-about.component.ts[17, 1]: Exceeds maximum line length of 140

#### Expected behavior
No error.
tslint version 5.7.0 is not giving any error.  But the latest version is throwing the above error.

**Comment by harshi1991** [comment]
I guess I missed something. Sorry it is not an issue.
